core/src/main/scala/chisel3/package.scala

```diff
@@ -361,4 +361,6 @@ package object chisel3 {
 
   final val deprecatedMFCMessage =
     "this feature will not be supported as part of the migration to the MLIR-based FIRRTL Compiler (MFC). For more information about this migration, please see the Chisel ROADMAP.md."
+
+  final val deprecatedPublicAPIMsg = "APIs in chisel3.internal will no longer be public as of 3.6"
```

Comment:
```suggestion
  final val deprecatedPublicAPIMsg = "APIs in chisel3.internal are not intended to be public"
```
They'll just be deprecated in 3.6, not private until 5 so I think we just warn and don't mention a version for when they'll be removed.